vnext/Shared/Networking/WinRTHttpResource.cpp

```diff
@@ -653,8 +652,7 @@ void WinRTHttpResource::AddResponseHandler(shared_ptr<IResponseHandler> response
     client = HttpClient{redirFilter};
   } else {
     auto globalOrigin = GetRuntimeOptionString("Http.GlobalOrigin");
```

Comment:
We should add a public PropertyId, or a setter for ReactPropertyBag/ReactInstanceSettings to allow setting this option on a per instance basis.  If someone wanted to have different origins for different RN instances there would still be a synchronization issue here.

We still need the "legacy" GetRuntimeOptionString option for Office's non-abi code paths, but for public use this should all be done using ReactInstanceSettings.

Reply:
Can that be addressed in a follow-up PR? (I could use that to re-factor other property ID declarations that are done inefficiently).

This fix is required ASAP.

Reply:
Sure